wear/src/main/kotlin/io/homeassistant/companion/android/home/HomeActivity.kt

```diff
@@ -44,6 +46,7 @@ class HomeActivity :
 
     companion object {
         private const val EXTRA_FROM_ONBOARDING = "from_onboarding"
+        const val LAUNCH_MODE = "launch_mode"
```

Comment:
You could make this private if you apply the same logic of making a static function in this class that build the action that you use in the UI.

Something like `gerLauchAction`